cupy/_core/_routines_math.pyx

```diff
@@ -902,6 +902,24 @@ _angle = create_ufunc(
     ''')
 
 
+def _positive_boolean_error():
+    raise TypeError(
```

Comment:
Unlike `negative`, it would be just `DeprecationWarning` with the current NumPy.

Reply:
Okay, I'll fix it so! Also, Array API layer is to check a boolean input.